### PR TITLE
Update gdcm to 3.0.0

### PR DESCRIPTION
**Pull request by tfmoraes**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Updates GDCM to last upstream version (3.0.0). Also compiles with VTK support. I needed to use Ninja instead of Make, because it doesn't compile with Make. Turned off the GDCM Applications because they (seems) to no be ported to last VTK version.

### CODE REVIEW

**Comment by tfmoraes** [comment]
If ITK was build from source the linkage test pass correctly.

File: Formula/gdcm.rb [hunk 1]
```diff
@@ -1,21 +1,24 @@
 class Gdcm < Formula
   desc "Grassroots DICOM library and utilities for medical files"
   homepage "https://sourceforge.net/projects/gdcm/"
-  url "https://downloads.sourceforge.net/project/gdcm/gdcm%202.x/GDCM%202.8.9/gdcm-2.8.9.tar.gz"
-  sha256 "a2da88b7b3cbf9e76a9df3e89d06d057cca9ce54fc62fb059e04f47bf056b727"
+  url "https://github.com/malaterre/GDCM/archive/v3.0.0.tar.gz"
+  sha256 "3de524690102bfa3e9c3a81ff3f17733138183f76b7a5f7d072b20024a255680"
 
   bottle do
-    sha256 "2b35bf311cd1fc39e0fc1b73e7dfe665eb252a5dfc9e5c0c9895bfd62d852b80" => :mojave
```
Comment (fxcoudert) [inline Formula/gdcm.rb]: Do not touch the bottle block.

**Review by fxcoudert — CHANGES_REQUESTED** [review]
Make the requested change, and squash everything into a single commit for `gdcm.rb`.
Then add a second commit, bumping the `revision` of `itk.rb`.

File: Formula/gdcm.rb (hunk 1 above)
Comment (tfmoraes) [reply Formula/gdcm.rb]: Ok

**Comment by tfmoraes** [comment]
Done, @fxcoudert !

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**